CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing to the Project
+
+This document contains and defines the rules that have to be followed by any
+contributor to the project, in order for any change to be merged into the
+stable branches.
+
+## Workflow
+
+The workflow followed is waterfall. In short, a contributor will target his 
```

Comment:
`waterflow`. The `waterfall` development model is something else :)

Reply:
Done